src/partials/warning-modal.html

```
@@ -0,0 +1,45 @@
+<!-- Warning Modal -->
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)